### DISCUSSION TITLE
Unchecked check marks are unclear

### DISCUSSION THREAD

**Discussion by Dieterbe** [iSoron/uhabits#1438] [Help & FAQ]
## Pre-submission checklist
- [x] I am submitting a bug report, not a feature request.
- [x] I have searched for similar issues, but did not find any matches.

## Description
While https://github.com/iSoron/uhabits/discussions/689 explains when unchecked checkmarks appear, it does not describe what they mean or what their purpose is.

## Steps to reproduce
1. Open the app and try to figure out what the unchecked marks are for 

## System information
 - Google Pixel 6
 - Android 12
 - App version: 2.0.3

**Comment by hiqua** [comment]
I'm not sure what you're missing from the answer of "What are unfilled checkmarks and why do they keep appearing?" in your link, do you have more specific questions that are not already answered?

**Comment by Dieterbe** [comment]
The faq entry describes that they are automatically added and when. It does not describe what their purpose is.

Specifically it says the app automatically adds them for non-daily habits on other days after adding a manual mark. But why ? What does it convey ?
Is the goal to prevent or discourage from doing a habit too often ? (Seems no, yes , respectively as I can still convert to a normal checkmark, presumably after doing the habit even if the day already has the auto-checkmark)

I think the dissonance is coming from the fact that checkmarks normally convey completed acts but here they mean the opposite. Perhaps a different icon might make more sense.

**Reply by iSoron** [reply]
Hi @Dieterbe, their purpose is to indicate that you don't need to do the habit that day (based on its frequency). For example, if you have a weekly habit and you complete it on Saturday, the app will add hollow checkmarks from Sunday through Friday to indicate that you're already done for the week.

**Comment by exanrauzer** [comment]
I am not sure if this is unclear. Unfilled checkmark is one of the key feature of this app. And the icon is well suited my opinion.

**Comment by LordJon** [comment]
I get the logic behind the confusion, but it's not something that happened to me. I agree with Exanrauzer, for me the icon makes perfect sense since the checkmark is a 'positive icon' which fits to an activity that already has been done. The difference in colour and brightness is an indicator that it was done in the past. To me it makes perfect sense.

Maybe more explanation is required in the FAQ so everyone gets the idea behind it?
